examples/README.md

```diff
@@ -9,3 +9,4 @@ Examples with `_codec` show how to use a custom codec. Examples with `_import_ho
 - `table` - Shows how you can easily generate an HTML table from data
 - `custom_components` - Shows how you can use custom components
 - `props` - Shows some advanced props usage
+- `custom_elements` - Shows how you can use custom HTML elements
```

Comment:
Maybe we can link to the [MDN page](https://developer.mozilla.org/en-US/docs/Web/API/Web_components/Using_custom_elements) on custom elements here?